corehq/apps/users/templates/users/web_users.html

```diff
@@ -1,364 +1,698 @@
-{% extends "users/base_template.html" %}
```

Comment:
A lot of the changes in this file are the result of changing web_users.b3.html to web_users.html  It's probably easier to review this one commit by commit (ignoring 9eae33a) https://github.com/dimagi/commcare-hq/commits/web-user-requests/corehq/apps/users/templates/users/web_users.html